build.gradle

```diff
@@ -667,9 +678,9 @@ def extensionCreate(dtype) {
 }
 def extensionMap(dtype) {
     if (dtype != "T") {
-        return "    = NDArray.${dtype.toLowerCase()}Factory.zeros(*shape().toIntArray()).fillLinear { f(this.get${dtype}(it)) }\n"
+        return "    = NDArray.createLinear(*shape().toIntArray(), filler={ f(this.get${dtype}(it)) } )\n"
     } else {
-        return "    = NDArray.createGeneric<T>(*shape().toIntArray(), filler = { f(this.getGeneric(*it)) })\n"
+        return "    = NDArray.createGeneric(*shape().toIntArray(), filler = { f(this.getGeneric(*it)) })\n"
```

Comment:
Why not use your optimized `createLinear` in this branch as well?

Reply:
Good point.  I'll change that.